packages/hyperflux/src/media/WebRTCPeerConnection.tsx

```diff
@@ -23,6 +23,7 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
 Infinite Reality Engine. All Rights Reserved.
 */
 
+import hark from 'hark'
```

Comment:
Hark is a front-end module, so if we can avoid putting this in hyperflux that'd be great.